template/nuxt.config.ts

```diff
@@ -52,10 +52,8 @@ module.exports = {
    * Extra Nuxt modules
    */
   'modules': [
-    'nuxt-babel',
-    'nuxt-imagemin',
+    // 'nuxt-imagemin',
```

Comment:
If this module is needed, explain the purpose of commenting.
If not, then remove it.

Reply:
Oops, yeap. This is still required. Good catch.